### DISCUSSION TITLE
CephFS mirroring:  failed to bootstrap peer

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#9030] [Q&A] [answered]
I followed Rook Docs about cephFS mirroring I created a secret that contains bootstrap peer token and added it to the cephFS Crd of the first cluster, but rook operator keeps throwing this error :   

```rook Error EINVAL: failed to bootstrap peer: exit status 22```

here is the command I used to create bootstrap peer token :
  
```ceph fs snapshot mirror peer_bootstrap create backup_fs client.mirror_remote site-remote```

**Comment by leseb** [comment, marked as answer]
I'm closing this in favor of https://github.com/rook/rook/issues/9045, let's use issues for bugs instead of discussion.
